2022-01-27 01:27:50,516 INFO    MainThread:16052 [wandb_setup.py:_flush():71] setting env: {}
2022-01-27 01:27:50,516 INFO    MainThread:16052 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-27 01:27:50,516 INFO    MainThread:16052 [wandb_init.py:_log_setup():357] Logging user logs to /home/psh/PycharmProjects/DTC_57/code/wandb/run-20220127_012750-3olpi199/logs/debug.log
2022-01-27 01:27:50,516 INFO    MainThread:16052 [wandb_init.py:_log_setup():358] Logging internal logs to /home/psh/PycharmProjects/DTC_57/code/wandb/run-20220127_012750-3olpi199/logs/debug-internal.log
2022-01-27 01:27:50,516 INFO    MainThread:16052 [wandb_init.py:init():390] calling init triggers
2022-01-27 01:27:50,516 INFO    MainThread:16052 [wandb_init.py:init():397] wandb.init called with sweep_config: {}
config: {'mdoel': 'DTC', 'server': 57, 'image size': '112,112,80'}
2022-01-27 01:27:50,516 INFO    MainThread:16052 [wandb_init.py:init():435] starting backend
2022-01-27 01:27:50,516 INFO    MainThread:16052 [backend.py:_multiprocessing_setup():96] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-27 01:27:50,526 INFO    MainThread:16052 [backend.py:ensure_launched():198] starting backend process...
2022-01-27 01:27:50,547 INFO    MainThread:16052 [backend.py:ensure_launched():204] started backend process with pid: 16068
2022-01-27 01:27:50,548 INFO    MainThread:16052 [wandb_init.py:init():444] backend started and connected
2022-01-27 01:27:50,549 INFO    MainThread:16052 [wandb_init.py:init():503] updated telemetry
2022-01-27 01:27:50,550 INFO    MainThread:16052 [wandb_init.py:init():533] communicating current version
2022-01-27 01:27:51,204 INFO    MainThread:16052 [wandb_init.py:init():538] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-01-27 01:27:51,205 INFO    MainThread:16052 [wandb_init.py:init():548] communicating run to backend with 30 second timeout
2022-01-27 01:27:51,473 INFO    MainThread:16052 [wandb_init.py:init():576] starting run threads in backend
2022-01-27 01:27:56,007 INFO    MainThread:16052 [wandb_run.py:_console_start():1693] atexit reg
2022-01-27 01:27:56,008 INFO    MainThread:16052 [wandb_run.py:_redirect():1567] redirect: SettingsConsole.REDIRECT
2022-01-27 01:27:56,008 INFO    MainThread:16052 [wandb_run.py:_redirect():1572] Redirecting console.
2022-01-27 01:27:56,010 INFO    MainThread:16052 [wandb_run.py:_redirect():1628] Redirects installed.
2022-01-27 01:27:56,010 INFO    MainThread:16052 [wandb_init.py:init():603] run started, returning control to user process
2022-01-27 01:27:57,325 INFO    MainThread:16052 [wandb_run.py:_atexit_cleanup():1663] got exitcode: 1
2022-01-27 01:27:57,325 INFO    MainThread:16052 [wandb_run.py:_restore():1635] restore
2022-01-27 01:27:59,532 INFO    MainThread:16052 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 586
  total_bytes: 586
}

2022-01-27 01:27:59,822 INFO    MainThread:16052 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 586
  total_bytes: 586
}

2022-01-27 01:28:00,506 INFO    MainThread:16052 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 586
  total_bytes: 6901
}

2022-01-27 01:28:00,608 INFO    MainThread:16052 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 586
  total_bytes: 8611
}

2022-01-27 01:28:00,721 INFO    MainThread:16052 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 586
  total_bytes: 8611
}

2022-01-27 01:28:00,823 INFO    MainThread:16052 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8611
  total_bytes: 8611
}

2022-01-27 01:28:00,925 INFO    MainThread:16052 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8611
  total_bytes: 8611
}

2022-01-27 01:28:01,027 INFO    MainThread:16052 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8611
  total_bytes: 8611
}

2022-01-27 01:28:01,129 INFO    MainThread:16052 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8611
  total_bytes: 8611
}

2022-01-27 01:28:01,231 INFO    MainThread:16052 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8611
  total_bytes: 8611
}

2022-01-27 01:28:01,507 INFO    MainThread:16052 [wandb_run.py:_wait_for_finish():1793] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8611
  total_bytes: 8611
}

2022-01-27 01:28:02,008 INFO    MainThread:16052 [wandb_run.py:_wait_for_finish():1793] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 8611
  total_bytes: 8611
}
local_info {
}

2022-01-27 01:28:03,151 INFO    MainThread:16052 [wandb_run.py:_append_files():2061] logging synced files
2022-01-28 10:33:22,688 INFO    MainThread:18504 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 0
2022-01-28 10:33:22,690 INFO    MainThread:18504 [wandb_run.py:_restore():1758] restore
2022-01-28 10:33:24,958 INFO    MainThread:18504 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 596
  total_bytes: 596
}

2022-01-28 10:33:25,219 INFO    MainThread:18504 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 596
  total_bytes: 596
}

2022-01-28 10:33:25,504 INFO    MainThread:18504 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 596
  total_bytes: 596
}

2022-01-28 10:33:26,409 INFO    MainThread:18504 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 596
  total_bytes: 5349
}

2022-01-28 10:33:26,515 INFO    MainThread:18504 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 596
  total_bytes: 5349
}

2022-01-28 10:33:26,641 INFO    MainThread:18504 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 596
  total_bytes: 5349
}

2022-01-28 10:33:26,757 INFO    MainThread:18504 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2067
  total_bytes: 5349
}

2022-01-28 10:33:26,862 INFO    MainThread:18504 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5349
  total_bytes: 5349
}

2022-01-28 10:33:26,967 INFO    MainThread:18504 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5349
  total_bytes: 5349
}

2022-01-28 10:33:27,077 INFO    MainThread:18504 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5349
  total_bytes: 5349
}

2022-01-28 10:33:27,182 INFO    MainThread:18504 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5349
  total_bytes: 5349
}

2022-01-28 10:33:27,287 INFO    MainThread:18504 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5349
  total_bytes: 5349
}

